compiler/hash-typecheck/src/diagnostics/error.rs

```diff
@@ -1225,6 +1228,30 @@ impl<'tc> From<TcErrorWithStorage<'tc>> for Report {
                     )));
                 }
             }
+            TcError::MethodNotAMemberOfTrait { trt_def_term_id, member, name } => {
+                builder.with_error_code(HashErrorCode::MethodNotAMemberOfTrait).with_message(
+                    format!(
+                        "method `{name}` is not a member of trait `{}`",
+                        trt_def_term_id.for_formatting_with_opts(
```

Comment:
Should say `function`, unless it should also check presence of `self`.

Reply:
But it is a method associated with the trait?

Reply:
Well, even static functions are called methods ? If so then okay.

Reply:
Yes sir